packages/app/src/Const.ts

```diff
@@ -178,4 +178,4 @@ export const MagnetRegex = /(magnet:[\S]+)/i;
  * Wavlake embed regex
  */
 export const WavlakeRegex =
-  /(?!zine\.wavlake\.com)(?:player\.|www\.)?wavlake\.com\/(?:(?:track|album)\/[a-f0-9]{8}(?:-[a-f0-9]{4}){3}-[a-f0-9]{12}|[a-z-]+)$/i;
+  /https?:\/\/(?:player\.|www\.)?wavlake\.com\/(?!top|new)(?:(?:track|album)\/[a-f0-9]{8}(?:-[a-f0-9]{4}){3}-[a-f0-9]{12}|[a-z-]+)/i;
```

Comment:
https://www.wavlake.com/artists renders a broken embed. Could you just make sure the link contains `track` or `album` instead?

```suggestion
  /https?:\/\/(player\.|www\.)?wavlake\.com\/(track|album)\/([a-f0-9]{8}(-[a-f0-9]{4}){3}-[a-f0-9]{12}|[a-z-]+)/i;
```

Reply:
Problem is our individual artists pages need to be matched too and the patterns for those are like `wavlake.com/artist-name`. We could reject match on `artists` path as well to be safe

Reply:
ah damn. that's means you'll have to update the regex every time you add a new path.

Reply:
Exactly. It's not ideal, but we don't have new paths pop-up very often. What do you think of this latest version? Covers all the existing cases (and a few that I don't see why someone would share but just in case).